content/docs/for-buildpack-authors/concepts/layer.md

```diff
@@ -3,8 +3,12 @@ title="What is a buildpack dependency layer?"
 weight=99
 +++
 
+`Dependency layers` are semantically meaningful layers that are contributed by one or more buildpacks during the building phase, one for each dependency.
```

Comment:
Should this say `Dependency layers` or, more simply, `Layers`?

Reply:
Don't we have different layers? if not, then why the title reads as "dependency layers"?

Reply:
We might first introduce the concept of an OCI image layer - see [here](https://github.com/opencontainers/image-spec/blob/main/layer.md) for a technical description which we might water down a little. It could be thought of as a filesystem changeset and some accompanying metadata. The ordering of layers usually matters for OCI images more generally but for buildpack contributed layers it does not (this is part of the beauty of our spec).

Then we might talk about the different kinds of layers you can have when building with buildpacks.
* There are base image layers (build and run), which are the layers that make up the underlying OS for the build-time container and the runtime container, respectively.
* Then you have buildpack-contributed dependency layers, which contain application dependencies (think: your language runtime & libraries).
* Then you have the application layer(s), which are usually some sort of transformation of the original source code that was provided (such as a compiled executable). We don't talk about "slices" here but we could link to it, it is a way to divide your application into many layers in order to make updates faster.
* Finally you have CNB-contributed layers, such as the SBOM, the launcher executable, and some configuration for the launcher.

Seeing as how this is buildpack author focused, we don't need to dwell on the first and last type of layer. But we can mention them to provide context.

The main point for buildpack authors is that by keeping layers small they can make them re-usable and composable. So I could have one layer that provides my language runtime and another that provides my package manager. The classic example is node (language) with either yarn or npm. This reduces duplication in what buildpack authors need to maintain.

Reply:
added